### PR TITLE
460 edited volume series

### LINKED ISSUE #460: Don't set type "EditedVolume" when "Series" is given
There are currently 875 resources that are of type `Series` and `EditedVolume`, example: http://lobid.org/resources/HT001232755#!

This doesn't make sense, we should only set type `EditedVolume` in these cases. 

(This issue has low priority.)

### PR DESCRIPTION
**Pull request by ChristophEwertowski**
Resolves #460

### CODE REVIEW

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 1]
```diff
@@ -710,6 +717,25 @@
 				<sanitizeUrl></sanitizeUrl>
 			</data>
 		</combine>
+		<combine name="@typeOnly" value="${a}">
+			<if>
+				<none>
+					<data source="052.">
+						<!-- <regexp match="........?|^.........?|^..........?"/> -->
+						<regexp match="^........([dcwesmbqfaghtz])|^.........([dcwesmbqfaghtz])|^..........([dcwesmbqfaghtz])"/>
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: is this comment necessary? If not, better remove it.

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 2]
```diff
@@ -1770,12 +1795,25 @@
...
+			<choose name="a">
+				<data source="9[01234][27][-a][-1].f" name="a">
+					<regexp match="^Aufsatzsammlung" format="$[ns-lobid-vocab]EditedVolume"/>
+				</data>
+				<data source="9[01234][27][-a][-1].9" name="a">
+					<regexp match="4143413" format="$[ns-lobid-vocab]EditedVolume"/>
+				</data>
+			</choose>
+		</combine>
 		<!-- "Verweisungsformen" literals -->
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: Seems to be a reduplication of the code at line 720. Is this not redundant?

File: src/main/resources/morph-hbz01-to-lobid.xml (hunk 1 above)
Comment (ChristophEwertowski) [inline src/main/resources/morph-hbz01-to-lobid.xml]: Removed in new push.

File: src/main/resources/morph-hbz01-to-lobid.xml (hunk 2 above)
Comment (ChristophEwertowski) [inline src/main/resources/morph-hbz01-to-lobid.xml]: It was redundant.

**Comment by dr0i** [comment]
Deployed to prodcution, closing.

**Pull request merged**